### PR TITLE
Update docs landing page

### PR DESCRIPTION
**Pull request by cliffckerr**
Old:
image

New:
image

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,14 +1,13 @@
 Starsim
 =======
 
+**WARNING! Starsim is still in the early stages of development. It is being shared solely for transparency and to facilitate collaborative development. It is not ready to be used for real research or policy questions.**
```
Comment (JSchripsema-IDM) [inline README.rst:4]: You can format this as a warning box and not just bold text by using the following syntax:

```
.. warning::
   Starsim is still in the early....
```

File: docs/index.rst [hunk 2]
```diff
@@ -1,9 +1,40 @@
+.. title:: Starsim
+
+
+.. include:: ../README.rst
```
Comment (JSchripsema-IDM) [inline docs/index.rst:4]: You should probably update the README to replace the sentence saying documentation isn't available to include a link to docs.idmod.org/projects/starsim

**Review by JSchripsema-IDM — APPROVED** [review]
Looks good, but I noted a couple issues to be updated.

File: docs/index.rst (hunk 2 above)
Comment (menriquez-IDM) [reply docs/index.rst:4]: there is currently a white IDM logo that is only visible if you are using the Dark theme:
**Ligth Mode:**
image

**Dark Mode:**
image

maybe you want to include a fix for this issue as well.

Comment (JSchripsema-IDM) [reply docs/index.rst:4]: Good catch! I can send the logo file if you need it.

File: README.rst (hunk 1 above)
Comment (cliffckerr) [reply README.rst]: That renders nicely in the docs:
image

But unfortunately GitHub doesn't parse that directive:
image

File: docs/index.rst (hunk 2 above)
Comment (cliffckerr) [reply docs/index.rst:4]: Nice catch! Fixed

**Pull request merged**